src/main/java/pl/ismop/web/client/dap/measurement/MeasurementService.java

```diff
@@ -19,15 +19,13 @@ void getMeasurements(@PathParam("timelineIds") String timelineIds,
 						 MethodCallback<MeasurementsResponse> callback);
 
 	@GET
-	@Path("measurements?timeline_id={timelineIds}&time_from={timeFrom}&time_to={timeTo}&limit=last")
+	@Path("measurements?timeline_id={timelineIds}&limit=last")
 	void getLastMeasurements(@PathParam("timelineIds") String timelineIds,
```

Comment:
`timeTo` should not be removed in this query. It is used e.g. in fibre view to show last measurement for specific point in time.